src/Client/Updater.php

```diff
@@ -634,4 +638,51 @@ public function download(string $target, ...$extra): PromiseInterface
         return $this->repoFileFetcher->fetchTarget($target, $length, ...$extra)
             ->then($verify);
     }
+
+    /**
+     * @param string $target
+     *   The path of the target file. Needs to be known to the most recent
+     *   targets metadata downloaded in ::refresh().
+     */
+    private function getMetadataForTarget(string $target, SnapshotMetadata $snapshotMetadata, ?TargetsMetadata $targetsMetadata = null): ?TargetsMetadata
```

Comment:
The other two parameters need to be documented here as well (plus a description).

Reply:
fixed